plugin/src/main/java/net/dzikoysk/funnyguilds/shared/TimeDivision.java

```diff
@@ -0,0 +1,96 @@
+package net.dzikoysk.funnyguilds.shared;
+
+import java.util.Arrays;
+
+public enum TimeDivision {
```

Comment:
nie no, weź to do configu wywal jeśli chcesz takie rzeczy wspierać. ale imo wydaje mi się, że 1d 4h 3m itd jest mocno uniwersalne i wystarczające.

Reply:
Jest już konwersacja na to wiec no https://github.com/FunnyGuilds/FunnyGuilds/pull/2114#discussion_r953669185